src/main/java/org/sagebionetworks/web/client/widget/table/modal/fileview/CreateTableViewWizardStep1ViewImpl.java

```diff
@@ -42,7 +42,6 @@
 	public CreateTableViewWizardStep1ViewImpl(Binder binder, FileViewOptions viewOptions, CookieProvider cookies) {
 		widget = binder.createAndBindUi(this);
 		this.viewOptions = viewOptions;
-		viewOptions.configure();
 		viewOptionsContainer.add(viewOptions);
```

Comment:
viewOptions.configure() unnecessary?

Reply:
Yes, the only thing it did was check experimental mode.